meeting_guide/migrations/0016_alter_group_gso_number_alter_meeting_area_and_more.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.2.5 on 2023-10-02 20:31
+
+from django.db import migrations, models
+import modelcluster.fields
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("meeting_guide", "0015_auto_20211002_1711"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="group",
+            name="gso_number",
+            field=models.CharField(
+                blank=True,
+                help_text="General Service Office (GSO) number, if applicable.",
+                max_length=10,
+                null=True,
+                verbose_name="GSO Number",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="meeting",
+            name="area",
+            field=models.CharField(blank=True, max_length=30),
```

Comment:
Increased max_length to 30 for the area CharField. Some areas (other fellowships) can have longer names. Hoping this doesn't break anything major 🤞

Reply:
I'm trying to remember if there was a reason for the lower number, but let's go with it.

Reply:
In this case, `area` is a two-digit number. AA's fellowship is broken up into Areas around the world:
https://www.aa.org/sites/default/files/literature/smf-146_AreaMapUSCAN_EN_1121.pdf

Pennsylvania has two Areas, 59 and 60, for the east and the west half of the state. All of Alberta is 78. Within each Area there are districts. There isn't a need to increase the size here... 10 was being generous! :) Some `help_text` would be a good idea, to explain this.

Reply:
Well that makes a bunch of sense! I didn't realize it was by area number. In CA we have a names, no numbers. I'll revert this with some help text :)

Reply:
![image](https://github.com/code4recovery/wagtail-meeting-guide/assets/4743971/501fca6c-ecee-4775-851d-b0c909bf9050)

Reverted back to 10 chars. Added help text with the link you commented.